cscs-checks/prgenv/helloworld.py

```diff
@@ -137,7 +137,7 @@ def __init__(self, lang, linkage):
         self.sourcepath += '_openmp.' + lang
         self.descr += ' OpenMP ' + str.capitalize(linkage)
         self.prgenv_flags = {
-            'PrgEnv-cray': self.cray_omp_flags(lang),
+            'PrgEnv-cray': ['-homp' if lang == 'F90' else '-fopenmp'],
```

Comment:
Why don't you use the `cray_omp_flags`? It is supposed to do what you are doing here.

Reply:
i removed the function, it's easier to read inlined